py/torch_tensorrt/dynamo/backend/backends.py

```diff
@@ -89,22 +106,37 @@ def _pretraced_backend(
             # Remove detach nodes
             remove_detach(gm, settings)
 
+            complexInputIndices = []
+            for i, torch_input in enumerate(torch_inputs):
+                if torch_inputs[i].dtype == torch.complex64:
+                    complexInputIndices.append(i)
+                    torch_input_real = torch_inputs[i].real
+                    torch_input_imaginary = torch_inputs[i].imag
+                    torch_inputs[i] = torch.stack(
+                        (torch_input_real, torch_input_imaginary), dim=-1
+                    )
+
             # Invoke AOTAutograd to translate operators to aten
-            gm = aot_export_joint_simple(
-                gm,
-                sample_inputs,
-                trace_joint=False,
-                decompositions=get_decompositions(
-                    settings.enable_experimental_decompositions
-                ),
-            )
+            if settings.use_aot_joint_export:
+                gm = aot_export_joint_simple(
+                    gm,
+                    sample_inputs,
+                    trace_joint=False,
+                    decompositions=get_decompositions(
+                        settings.enable_experimental_decompositions
+                    ),
+                )
 
             logger.debug("Post-AOT Autograd graph:\n" + str(gm.graph))
 
             gm = post_lowering(gm, settings)
 
             logger.debug("Lowered Input graph:\n " + str(gm.graph))
 
+            complex_nodes = find_complex_nodes(gm)
```

Comment:
Why isnt this part of lowering?

Reply:
Need to get the complex_nodes before I do the lowering pass of `replace_complex_placeholder_to_tuple()`. Can put this in` replace_complex_placeholder_to_tuple` lowering pass, but since its a util function and pertaining more to `modify_complex_nodes` I put this in converter_utils

Reply:
I mean seems to me these 3-4 lines are a pass that can be added to lowering